.eslintrc.js

```diff
@@ -166,7 +166,13 @@ module.exports = {
     'no-undef-init': 0,
     'no-undef': 1,
     'no-undefined': 0,
-    'no-unused-vars': [1, { args: 'none' }],
+
+    'no-unused-vars': 'off',
```

Comment:
while working with TypeScript, original `no-unused-vars` must be disabled as it can report incorrect errors. [more info](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unused-vars.md#how-to-use)